src/main/java/view/UploadView.java

```diff
@@ -74,8 +84,11 @@ public void propertyChange(PropertyChangeEvent evt) {
         String errorMessage = uploadState.getErrorMessage();
         if (episode != null) {
             status.setText(String.format("Successfully uploaded file for %s", episode.getTitle()));
+            transcribeController.execute(episode);
```

Comment:
Hmm, should it be here, or in the submit button action listener, after executing the upload use case?

Reply:
You're absolutely right it should be in the submit button action listener, I only added it here to make sure it's being executed after the upload use case is successfully executed

Reply:
Actually, I think you're right. We only want to transcribe if the upload use case succeeds.